tests/testthat/test-write-permissions.R

```diff
@@ -0,0 +1,14 @@
+
+context("error without write permissions")
+
+test_that("test failed write errors for saveWorkbook", {
+  tempFile <- fs::file_temp(ext = "xlsx")
+  fs::file_create(tempFile, mode = "a=rx")
+
+  wb <- createWorkbook()
+  addWorksheet(wb, "name")
+
+  expect_warning(write.xlsx(x = cars, file = tempFile, overwrite = TRUE))
```

Comment:
Maybe search for `"Permission denied"`?

Reply:
Great idea. I'll add that in.

Reply:
Implemented in ed1f0d8dd39d6e536ff8902b071d6c3b78cefe78